frontend/src/charts/trendsChart/Index.tsx

```diff
@@ -274,16 +278,58 @@ export function TrendsChart({
             isSkinny={isSkinny}
             chartWidth={width}
             breakdownVar={breakdownVar}
-            legendId={`legend-filter-label-${axisConfig.type}-${
-              isCompareCard ? '2' : '1'
-            }`}
+            legendId={`legend-filter-label-${axisConfig.type}-${isCompareCard ? '2' : '1'
+              }`}
```

Comment:
you probably need to keep this isCompareCard logic so that in compare mode the two cards will have different `id` values; otherwise it's an accessibility violation